src/components/body/index.js

```diff
@@ -36,8 +37,10 @@ function getElementComponent(type) {
     case 'embeddedCode':
     case 'embeddedcode':
       return null
+    case 'small-image':
+    case 'image-link':
```

Comment:
Is `image-link` ok to use `HorizontalImage`?

Reply:
It seems that we do not have the type `image-link` actually. We use `embedded-code` to render the element for `image-link` in keystone editor. If we add a image link type in the future. we just have to make the data format of `image-link` and `horizontal-image` the same.

Reply:
I'll add this into comment as TODO